pkg/workflows/wasm/sdk.go

```diff
@@ -1,26 +1,53 @@
 package wasm
 
 import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"unsafe"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/custmsg"
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
 	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
+	wasmpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/pb"
+)
+
+const (
```

Comment:
these are also defined in [pkg/capabilities/events/events.go](https://github.com/smartcontractkit/chainlink-common/blob/ceeb47375a5679524c088fe04a66105efc1f7a71/pkg/capabilities/events/events.go#L18). Is it possible to avoid duplication here?